test/Microsoft.ML.Tests/Transformers/TextFeaturizerTests.cs

```diff
@@ -197,5 +197,44 @@ public void NgramWorkout()
             CheckEquality("Text", "ngrams.tsv");
             Done();
         }
+
+        [Fact]
+        public void LdaWorkout()
+        {
```

Comment:
Perhaps for now forego the baseline test, and `Skip` this test, but still have a test where you try to apply the transform, iterate over a few rows, make sure the outputs has the right number of outputs, etc., at least for now. But we should write an issue too -- there's something deeper that's wrong that seems to be happening here.

Reply:
Not in your code, mind, but in the original code.

---
In reply to: [219981263](https://github.com/dotnet/machinelearning/pull/972#discussion_r219981263) [](ancestors = 219981263)

Reply:
Thanks @tfinley@gmail.com, disabled the full output checking the test. Also, opened issue #1004 to track it.

---
In reply to: [219981354](https://github.com/dotnet/machinelearning/pull/972#discussion_r219981354) [](ancestors = 219981354,219981263)